ecs-cli/modules/cli/local/ps_app.go

```diff
@@ -70,9 +71,12 @@ func Ps(c *cli.Context) {
 
 func listContainers(c *cli.Context) ([]types.Container, error) {
 	if c.IsSet(flags.TaskDefinitionFile) {
+		file, err := filepath.Abs(c.String(flags.TaskDefinitionFile))
```

Comment:
can you also add this functionality at l.95 for `localproject.LocalInFileName`? (the last return statement)

Reply:
good catch -- fixed.